CHANGELOG.md

```diff
@@ -2,11 +2,16 @@
 
 ## [Release 2.16.2](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.16.2)
 
+### Bug Fixes
+
+- **AWS IoT**
+  - Added minimum sdk and target sdk version.
```

Comment:
Add a link to the build.gradle and AndroidManifest.xml and also specify the numbers 11 and 27 here.

Reply:
I don't see a point in specifying min sdk, since it didnt change, but did specify bumping 27 to 28.
Did not link the files, but mentioned them as having been edited.